examples/buttons/button-styled.py

```diff
@@ -0,0 +1,23 @@
+from tkinter import Tk, Button #importing Tk and button from tkinter
+from tkinter import ttk #importing ttk from tkinter that is used to style the tkinter widgets
+
+'''initilaizing tkinter window'''
+root=Tk()
+
+'''Adding Normal Button '''
+btn1 = Button(root, text = 'Normal Button')
+btn1.pack(pady=20) #pady means padding along y axis
```

Comment:
Comments like this are not useful and should be deleted. In fact, there is only one useful comment in this file. The code is clearly creating a button, and it is easy to google `tkinter pady`. If there is too much comments, nobody will read them.